src/test/rustdoc-gui/jump-to-def-background.goml

```diff
@@ -0,0 +1,23 @@
+// We check the background color on the jump to definition links in the source code page.
+goto: file://|DOC_PATH|/src/link_to_definition/lib.rs.html
+
+// Set the theme to dark.
+local-storage: {"rustdoc-theme": "dark", "rustdoc-preferred-dark-theme": "dark", "rustdoc-use-system-theme": "false"}
+// We reload the page so the local storage settings are being used.
+reload:
+
+assert-css: ("body.source .example-wrap pre.rust a", {"background-color": "rgb(51, 51, 51)"}, ALL)
```

Comment:
It'd probably be a good idea to add a check that there are any links at all, just in case the bootstrap check isn't working.

Reply:
It fails if no element matches the given selector. :)

Reply:
Ah, that makes sense 👍